content/join/index.md

```diff
@@ -7,6 +7,22 @@ menu:
     weight: 1
 ---
 
+# Contributing to the OPI Project
+
+Please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSdYilTsg74EZCh2_5G6dPtNd0z3EQv4Pc63brUUpM1XMQkzMg/viewform?usp=sf_link)
+if you are interested in contributing to the Open Programmable Infrastructure
+Project. Once you have filled out the participation form you will be added to
```

Comment:
We should change the text here as well. Putting check marks in the spreadsheet isn't working well. We should say the form will get you added to mailing list and invited to the main meeting. Maybe point to the subgroups page in github for more info the sub groups. Join the slack to connect to the sub group leader to join a sub group??